docs/usage/validation_errors.md

```diff
@@ -177,6 +177,36 @@ except ValidationError as exc:
     #> 'callable_type'
 ```
 
+## `dataclass_exact_type`
+
+This error is raised when validating a dataclass with `strict=True`:
```

Comment:
Can we improve this? I think it is not clear enough

Reply:
ok.

Reply:
done.